app/controllers/pay_by_account_controller.rb

```diff
@@ -0,0 +1,35 @@
+class PayByAccountController < ApplicationController
+  rescue_from GlimrApiClient::PayByAccount::AccountNotFound,
```

Comment:
I think you can combine the rescue_from calls into a single call with multiple exception classes http://api.rubyonrails.org/v5.0/classes/ActiveSupport/Rescuable/ClassMethods.html